src/tike/operators/cupy/convolution.py

```diff
@@ -1,21 +1,133 @@
+__author__ = "Daniel Ching, Viktor Nikitin"
+__copyright__ = "Copyright (c) 2020, UChicago Argonne, LLC."
+
 from importlib_resources import files
 
 import cupy as cp
```

Comment:
Does it make sense to use managed memory in cupy to handle arrays not fitting into the gpu memory? or it will make the code very-very slow?
```
pool = cp.cuda.MemoryPool(cp.cuda.malloc_managed)
cp.cuda.set_allocator(pool.malloc)
```

Reply:
I understand conceptually what CUDA unified memory is, but I don't know how to use it with CuPy because it is not well documented. Is it literally as easy as adding those two lines to the top of any script?